gallery.html

```diff
@@ -45,67 +45,67 @@ <h1>Gallery</h1>
 
             <div class="gallery">
                 <a class="row-span-2" href="https://loremflickr.com/1024/1024/?lock=50"> 
-                    <img src="https://loremflickr.com/300/300?lock=50" /> 
+                    <img src="https://loremflickr.com/300/600?lock=50" alt="Gallery image" /> 
                 </a>
 
                 <a href="https://loremflickr.com/1024/1024/?lock=55"> 
-                    <img src="https://loremflickr.com/300/600?lock=55" /> 
+                    <img src="https://loremflickr.com/300/300?lock=55" alt="Gallery image" /> 
                 </a>
 
                 <a href="https://loremflickr.com/1024/1024/?lock=85"> 
-                    <img src="https://loremflickr.com/300/300?lock=85" /> 
+                    <img src="https://loremflickr.com/300/300?lock=85" alt="Gallery image" /> 
                 </a>
 
                 <a href="https://loremflickr.com/1024/1024/?lock=864"> 
-                    <img src="https://loremflickr.com/300/300?lock=864" /> 
+                    <img src="https://loremflickr.com/300/300?lock=864" alt="Gallery image" /> 
                 </a>
 
                 <a href="https://loremflickr.com/1024/1024/?lock=22"> 
-                    <img src="https://loremflickr.com/300/300?lock=22" /> 
+                    <img src="https://loremflickr.com/300/300?lock=22" alt="Gallery image" /> 
                 </a>
 
                 <a href="https://loremflickr.com/1024/1024/?lock=10"> 
-                    <img src="https://loremflickr.com/300/300?lock=10" /> 
+                    <img src="https://loremflickr.com/300/300?lock=10" alt="Gallery image" /> 
                 </a>
 
                 <a href="https://loremflickr.com/1024/1024/?lock=1"> 
-                    <img src="https://loremflickr.com/300/300?lock=1" /> 
+                    <img src="https://loremflickr.com/300/300?lock=1" alt="Gallery image" /> 
                 </a>
                 
                 <a href="https://loremflickr.com/1024/1024/?lock=5"> 
-                    <img src="https://loremflickr.com/300/300?lock=5" /> 
+                    <img src="https://loremflickr.com/300/300?lock=5" alt="Gallery image" /> 
                 </a>
 
                 <a href="https://loremflickr.com/1024/1024/?lock=32"> 
-                    <img src="https://loremflickr.com/300/300?lock=32" /> 
+                    <img src="https://loremflickr.com/300/300?lock=32" alt="Gallery image" /> 
                 </a>
 
-                <a class="row-span-2" class="col-span-2" href="https://loremflickr.com/1024/1024/?lock=70"> 
```

Comment:
Multiple classes are indicated using space separated values in ONE class attribute.  You may not have multiple class attributes